OpenRA.Mods.Common/Traits/Attack/AttackGarrisoned.cs

```diff
@@ -93,7 +93,7 @@ public AttackGarrisoned(Actor self, AttackGarrisonedInfo info)
 			getArmaments = () => armaments;
 		}
 
-		public void PassengerEntered(Actor self, Actor passenger)
+		public void OnPassengerEntered(Actor self, Actor passenger)
```

Comment:
Since you're renaming this, should this be now explicitly implemented according to your recent craze?